apps/src/templates/census2017/CensusMap.jsx

```diff
@@ -215,7 +226,10 @@ export default class CensusMap extends Component {
     return infoWindowDom;
   };
 
-  handleTakeSurveyClick = (schoolDropdownOption) => {
+  handleTakeSurveyClick = (schoolDropdownOption, existingInaccuracy) => {
+    this.props.onShowExistingInaccuracyChange(existingInaccuracy);
+    this.props.onExistingInaccuracyChange(existingInaccuracy);
+
```

Comment:
It would make more sense to move this logic up into YourSchool since that is where `showExistingInaccuracy` and `exitingInaccuracy` live. It could be triggered by passing a separate `onInaccurcyReportClick` callback to CensusMap which would set `showExistingInaccuracy` and `exitingInaccuracy` correctly.

Reply:
Given that we still have to call `adjustScroll('form');`, we would still need a new function here too, wouldn't we?

Reply:
Maybe just pass the existingInaccuracy Boolean up to onSchoolChange and set the YourSchool state based on that. 

This component should be reporting what happened internally (report inaccuracy was clicked) and not what we expect the parent to do in response (show the checkbox and make it checked.)

Reply:
Ah of course.  and `adjustScroll` works better on the parent anyway.  Just committed changes to this.